### PR TITLE
fix contract bug and update scripts

### PR DESCRIPTION
**Pull request by vsszhang**
Main Update( current contract version)

fix contract bug:
- ReputationV1.sol withdraw function

add new content into contract
- pause() and unpause() control func in PoapFactory.sol
- burnOne(), _burnAClass(), getPoapIdentifier() in ZcloakPoap.sol

add new test script:
- RAC.test.js
- Reputation_multikeeper.test.js

### CODE REVIEW

File: test/Aggregator.test.js [hunk 1]
```diff
@@ -37,12 +39,13 @@ describe("SimpleAggregator Contract", function () {
     let keeper3;
     let keeper4;
     let keeper5;
+    let alith;
 
     let rHash;
```
Comment (dejavukong) [inline test/Aggregator.test.js]: requestHash? rootHash? better naming to be more read-friendly

File: test/Aggregator.test.js [hunk 2]
```diff
@@ -37,12 +39,13 @@ describe("SimpleAggregator Contract", function () {
     let keeper3;
     let keeper4;
     let keeper5;
+    let alith;
```
Comment (dejavukong) [inline test/Aggregator.test.js]: alith? alice?

File: contracts/ZcloakPoap.sol [hunk 3]
```diff
@@ -19,13 +19,24 @@ contract ZCloakPoap is ERC1155, Pausable, Ownable, Properties {
     mapping(uint128 => uint128) totalSupply;
     // poapId => userAddr => balance
     mapping(uint128 => mapping(address => uint256)) public totalBalanceOf;
+    /// poapIdentifier => userAddr => MyPoapId
+    /// @notice current for test
+    mapping(uint128 => mapping(address => uint256)) public myPoapId;
```
Comment (dejavukong) [inline contracts/ZcloakPoap.sol]: suggest: test functions should be added in test contracts.

File: test/Aggregator.test.js (hunk 1 above)
Comment (vsszhang) [reply test/Aggregator.test.js]: modify rHash to requestHash

File: test/Aggregator.test.js (hunk 2 above)
Comment (vsszhang) [reply test/Aggregator.test.js]: modify alith to keeper6

File: contracts/ZcloakPoap.sol (hunk 3 above)
Comment (vsszhang) [reply contracts/ZcloakPoap.sol]: transfer some test functions to mock contract and delete others

**Pull request merged**